shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ExplainStatementTestCase.java

```diff
@@ -22,5 +22,5 @@
 /**
  * Describe statement test case.
  */
-public final class DescribeStatementTestCase extends SQLParserTestCase {
+public final class ExplainStatementTestCase extends SQLParserTestCase {
```

Comment:
We do not have tests to assert its `SQLStatement` field currently, which is possible to be some community issues for our volunteers.

Reply:
> We do not have tests to assert its `SQLStatement` field currently, which is possible to be some community issues for our volunteers.

@tristaZero Great suggestion, I will create some community tasks later. 👍